inc/mailhandlers/digestmd5.php

```diff
@@ -0,0 +1,179 @@
+<?php
+// +-----------------------------------------------------------------------+ 
+// | Copyright (c) 2002-2003 Richard Heyes                                 | 
```

Comment:
All merged codes should be copyrighted to MyBB only being you are a contributor to it ...